src/parsers/sany.ts

```diff
@@ -248,7 +248,7 @@ export class SanyStdoutParser extends ProcessOutputHandler<SanyData> {
     }
 
     private tryParseModulePath(line: string) {
-        const rxJarPath = /^(.*)(?: \(jar:file:.*\))$/g;
+        const rxJarPath = /^(.*)(?: \(.*\))$/g;
```

Comment:
Does it make sense to change the name of the const to something other than `rxJarPath` given that the regex has been relaxed?

Reply:
Good point, updated, thanks